### PR TITLE
Adds some missing translations

### PR DESCRIPTION
**Pull request by Tiliavir**
I would suggest squashing this commit. Sorry for the mess, did it in the browser ...

@hed854 Could you check my french translations? even so I live very close to the border, I have no idea if that is correct xD I'll give you permissions on the repo, that will makes things easier I guess ;-)

### CODE REVIEW

File: i18n/fr.toml [hunk 1]
```diff
@@ -16,6 +16,15 @@ other = "Instructions"
 [components]
 other = "Composantes"
 
+[this-is-a]
+other = "C'est un"
```
Comment (hed854) [inline i18n/fr.toml]:
```suggestion
other = "Ceci est une"
```

File: i18n/fr.toml [hunk 2]
```diff
@@ -16,6 +16,15 @@ other = "Instructions"
...
+
+[component-based-recipe]
+other = "recette à base de composants"
+
+[fancy-talk]
+other = "fantaisie formulé pour servir un plat en morceaux"
```
Comment (hed854) [inline i18n/fr.toml]:
```suggestion
other = "formulation désignant la création d'un plat à partir de sous-recettes"
```

**Review by hed854 — COMMENTED** [review]
Added two corrections (as commit suggestions) - otherwise all OK for French translations :+1:

**Comment by Tiliavir** [comment]
@hed854 thank you very much!🎉

**Comment by seanlane** [comment]
Thanks!

**Pull request merged**